web/forms.py

```
@@ -1677,3 +1680,42 @@
+
+class PDFSubmissionForm(forms.ModelForm):
+    """Form for submitting PDF documents."""
+
+    pdf_type = forms.ModelChoiceField(
+        queryset=PDFType.objects.all(),
+        empty_label="Select PDF Type",
+        widget=forms.Select(
+            attrs={
+                "class": "bg-white dark:bg-gray-800 text-gray-900 dark:text-white border border-gray-300 "
+                "dark:border-gray-600 rounded-md p-2 focus:border-blue-500 focus:outline-none focus:ring-2 "
+                "focus:ring-blue-500 w-full"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)